tests/snappy_pipeline/workflows/test_workflows_somatic_gene_fusion_calling_processed_fastq.py

```diff
@@ -0,0 +1,404 @@
+# -*- coding: utf-8 -*-
+"""Tests for the somatic_gene_fusion_calling workflow module code"""
```

Comment:
Maybe just left this as last step, after all changes to the regular tests for `somatic_gene_fusion` are completed. For instance, the last portion of the code still commented.